synapse/lib/cell.py

```diff
@@ -226,6 +226,9 @@ async def promote(self):
     def getCellUser(self):
         return self.user.pack()
 
+    async def getCellVersionInfo(self):
```

Comment:
I'm not entirely sold on this API name. It feels like we have a bit of API sprawl going on that could be unified.
Information from ``getCellType()`` and ``getCellIden()`` could just as easily also be added to the data being returned by this API.

Reply:
``getCellInfo()`` for some basic info?

Reply:
Yeah `getCellInfo` seems like a good way to go to consolidate things.

Reply:
:+1: